From decdb9659c0dfb6ea8719b639ff3259fed0fa145 Mon Sep 17 00:00:00 2001
From: Andy Whitcroft <apw@canonical.com>
Date: Wed, 20 Apr 2011 12:08:05 +0100
Subject: [PATCH] DISABLE olpc

TRIGGER 13b325ae1aeee7f8602759057ed0d1f9 drivers/staging/olpc_dcon/olpc_dcon_xo_1.c

Signed-off-by: Andy Whitcroft <apw@canonical.com>
---
 debian.master/config/config.common.ports |    4 +++-
 drivers/staging/Makefile                 |    2 +-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/debian.master/config/config.common.ports b/debian.master/config/config.common.ports
index 8b13789..4574947 100644
--- a/debian.master/config/config.common.ports
+++ b/debian.master/config/config.common.ports
@@ -1 +1,3 @@
-
+#
+# Common config options automatically generated by splitconfig.pl
+#
diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index eb93012..208c215 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -20,7 +20,7 @@ obj-$(CONFIG_BRCM80211)		+= brcm80211/
 obj-$(CONFIG_RT2860)		+= rt2860/
 obj-$(CONFIG_RT2870)		+= rt2870/
 obj-$(CONFIG_COMEDI)		+= comedi/
-obj-$(CONFIG_FB_OLPC_DCON)	+= olpc_dcon/
+#obj-$(CONFIG_FB_OLPC_DCON)	+= olpc_dcon/
 obj-$(CONFIG_ASUS_OLED)		+= asus_oled/
 obj-$(CONFIG_PANEL)		+= panel/
 obj-$(CONFIG_R8187SE)		+= rtl8187se/
-- 
1.7.1

